sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -119,49 +130,53 @@ Once completed, set the values of the client ID, tenant ID, and client secret of
- - Recognizing common fields from the following form types using prebuilt models. These fields and metadata are returned in a collection of `RecognizedForm` objects.
-    - Sales receipts. See fields found on a receipt [here][service_recognize_receipt].
-    - Business cards. See fields found on a business card [here][service_recognize_business_cards].
-    - Invoices. See fields found on an invoice [here][service_recognize_invoice].
-    - Identity documents. See fields found on identity documents [here][service_recognize_identity_documents].
- - Recognizing form content, including tables, lines, words, and selection marks, without the need to train a model. Form content is returned in a collection of `FormPage` objects.
+|"prebuilt-layout" | Text extraction, selection marks, tables
+|"prebuilt-document" | Text extraction, selection marks, tables, key-value pairs and entities
+|"prebuilt-invoices" | Text extraction, selection marks, tables, and pre-trained fields and values pertaining to English invoices
+|"prebuilt-businessCard" | Text extraction and pre-trained fields and values pertaining to English business cards
+|"prebuilt-idDocument" | Text extraction and pre-trained fields and values pertaining to US driver licenses and international passports
+|"prebuilt-receipt" | Text extraction and pre-trained fields and values pertaining to English sales receipts
+|"{custom-model-id}" | Text extraction, selection marks, tables, labeled fields and values from your custom documents
 
-Sample code snippets are provided to illustrate using a FormRecognizerClient [here](#recognize-forms-using-a-custom-model "Recognize Forms Using a Custom Model").
+Sample code snippets are provided to illustrate using a DocumentAnalysisClient [here](#examples "Examples").
 
-### FormTrainingClient
-`FormTrainingClient` provides operations for:
+### DocumentModelAdministrationClient
+`DocumentModelAdministrationClient` provides operations for:
 
-- Training custom models without labels to recognize all fields and values found in your custom forms. A `CustomFormModel` is returned indicating the form types the model will recognize, and the fields it will extract for each form type. See the [service documentation][fr-train-without-labels] for a more detailed explanation.
-- Training custom models with labels to recognize specific fields, selection marks, tables, and values you specify by labeling your custom forms. A `CustomFormModel` is returned indicating the fields the model will extract, as well as the estimated accuracy for each field. See the [service documentation][fr-train-with-labels] for a more detailed explanation.
+- Building custom models to extract specific fields you specify by labeling your custom documents. A `DocumentModel` is returned indicating the document type the model will extract, as well as the estimated confidence for each field. See the [service documentation][fr-train-with-labels] for a more detailed explanation.
```

Comment:
extracet -> analyze?

Reply:
indicating the document type the model will extract -> indicating the document type **of** the model **that** will **analyze**